.gitignore

```diff
@@ -16,11 +16,14 @@ testbin/*
 
 # Kubernetes Generated files - skip generated files, except for vendored files
 
-!vendor/**/zz_generated.*
+# !vendor/**/zz_generated.*
```

Comment:
just remove the line and the comment above

Reply:
sg.